src/components/ContactUsForm/ContactUs.jsx

```diff
@@ -0,0 +1,135 @@
+import { useBackend } from '../../contexts/BackendContext';
+import { useState } from 'react';
+import { Flex, Text, Textarea, Stack, Box, Spacer, Button, ButtonGroup, Checkbox } from '@chakra-ui/react';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ButtonGroup' is defined but never used.

Reply:
remove variables that we are not using